internal/api/mocks/schema_service.go

```diff
@@ -18,6 +23,10 @@ type SchemaService struct {
 func (_m *SchemaService) CheckCompatibility(ctx context.Context, nsName string, schemaName string, compatibility string, data []byte) error {
 	ret := _m.Called(ctx, nsName, schemaName, compatibility, data)
 
+	if len(ret) == 0 {
+		panic("no return value specified for CheckCompatibility")
+	}
+
```

Comment:
can we return an error instead of panic, since this method returns an error anyways?